app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
A button related to an input field (a checkbox or a radio), which appearance will change if the field is checked.
In other words the view of a checkbox or radio field under the shape of a button.

Can you think of another name ?